build.gradle

```diff
@@ -74,7 +78,7 @@ dependencies {
   compileOnly 'com.fasterxml.jackson.core:jackson-databind:2.7.4'
   compileOnly 'com.fasterxml.jackson.module:jackson-module-afterburner:2.7.4'
   compileOnly 'com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.7.4'
-  compileOnly group: 'org.jruby', name: 'jruby-complete', version: '9.1.12.0'
+  compileOnly group: 'org.jruby', name: 'jruby-complete', version: '9.2.11.0'
```

Comment:
noticed this on another piece as well ... 
should compilation be done against the lowest JRuby (included in oldest LS supported) ?
as otherwise compatibility might break easily, if a newer JRuby method gets used.